Adding note about historical data for anomaly monitors (#27187)
DataDog/documentation: content/en/monitors/types/cloud_cost.md

| Cost Type | Description | Usage Examples |
| ---  | ----------- | ----------- |

> | Cost Anomalies | Detect anomalies by comparing current costs to historical data, using a defined lookback period. Incomplete days are excluded from analysis to ensure accuracy. Anomaly monitors will not evaluate without at least 4 months of cloud cost data, since historical data is required to train the algorithm. | Alert if 3 days from the past 30 days show significant cost anomalies compared to historical data. |

Reviewer: ```suggestion
| Cost Anomalies | Detect anomalies by comparing current costs to historical data, using a defined lookback period. Incomplete days are excluded from analysis to ensure accuracy. Anomaly monitors require at least 4 months of cloud cost data to evaluate since historical data is required to train the algorithm. | Alert if 3 days from the past 30 days show significant cost anomalies compared to historical data. |
```
